frontend/src/herd_view.tsx

```diff
@@ -4,55 +4,147 @@
  *       belonging to that herd.
  */
 import React from 'react'
-import {Link} from "react-router-dom";
+import { makeStyles } from '@material-ui/core/styles';
+import { AppBar, Box, Paper, Tab, Tabs } from '@material-ui/core';
+import { get } from '@app/communication';
+import { Herd, Individual } from '@app/data_context_global';
+import { HerdForm } from '@app/herdForm';
+import { useMessageContext } from '@app/message_context';
+import { useDataContext } from '@app/data_context';
+import { herdPedigree } from '@app/pedigree';
+import { PedigreeNetwork } from '@app/pedigree_plot';
+import { FilterTable } from '@app/filter_table';
+import { IndividualView } from './individual_view';
 
-import { get } from './communication';
-import { herdLabel } from '@app/data_context_global';
+const useStyles = makeStyles({
+  container: {
+    padding: "20px",
+  },
+  title: {
+    fontSize: '1.33em',
+    fontWeight: 'bold',
+  },
+  animalList: {
+    cursor: 'pointer',
+    '&:hover': {
+      color: 'blue',
+    }
+  },
+});
+
+interface TabPanelProps {
```

Comment:
Maybe you want `interface TabPanelProps extends React.DetailedHTMLProps<React.HTMLAttributes<HTMLDivElement>, HTMLDivElement> {` to support `...other` below. Haha I wish this type was not so unwieldy.

Reply:
good point, and yeah, they could have named it `React.ElementProps` imho xD